packages/react-dev-utils/FileSizeReporter.js

```diff
@@ -92,12 +92,12 @@ function printFileSizesAfterBuild(
     );
     console.log(
       chalk.yellow(
-        'Consider reducing it with code splitting: https://goo.gl/9VhYWB'
+        'Consider reducing it with code splitting: https://create-react-app.dev/docs/code-splitting/'
```

Comment:
I've created custom shortlinks for these. This one should be `https://cra.link/code-splitting`